src/ripple/app/ledger/Ledger.h

```diff
@@ -408,6 +408,7 @@ class Ledger final : public std::enable_shared_from_this<Ledger>,
     Fees fees_;
     Rules rules_;
     LedgerInfo info_;
+    beast::Journal j_ = beast::Journal(beast::Journal::getNullSink());
```

Comment:
I think it's safe to leave this unassigned in the class definition. It then gets assigned in the constructor.

Reply:
Sure. Fixed.